Students/Neacsu Ioana - Alina/Laborator5/src/lab5/Customer.java

```diff
@@ -0,0 +1,92 @@
+package lab5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Customer {
+    private String customerID;
+    private String name;
+    private String contactInfo;
+    private List<BankAccount> accounts;
+
+    public Customer() {
+        this.accounts = new ArrayList<>();
+    }
+    public Customer(String customerID, String name, String contactInfo){
+        this.name = name;
+        this.customerID = customerID;
+        this.contactInfo = contactInfo;
+        this.accounts = new ArrayList<>();
+    }
+    public BankAccount createNewAccount(String accountNumber, String accountType, float amount){
+        BankAccount newAcc = new BankAccount(this);
+        newAcc.setAccountNumber(accountNumber);
+        newAcc.setAccountType(accountType);
+        newAcc.setAmount(amount);
+        accounts.add(newAcc);
+        return newAcc;
+    }
+
+    public void newTransaction(BankStaff staff, String transactionID, BankAccount sourceAccount, BankAccount destinationAccount, TransactionType transactionType,float amount){
```

Comment:
cand ai asa multi parametri se obisnuieste una din urmatoarele:
1. pui cate unul pe linie
In cazul de fata ar fi fost ceva de genul
```
public void newTransaction(
   BankStaff staff, 
   String transactionID, 
   BankAccount sourceAccount, 
   BankAccount destinationAccount, 
   TransactionType transactionType,
   float amount
) {
```
3. ii bagi pe toti intr-o alta clasa si transmiti un singur obiect ca si param care-i contine pe toti. In cazul de fata ar fi fost `TransactionDetails` de exemplu

Reply:
chiar aveai deja obiectul.. `Transaction`